docs/content/config/index.md

```diff
@@ -84,6 +86,26 @@ more secure authentication like 2FA or WebAuthn.
 |   3   | `safely unguessable` and `very unguessable` passwords are allowed | 100.000.000                                       |
 |   4   | Only `very unguessable` passwords are allowed                     | 10.000.000.000                                    |
 
+### LDAP
+
+HedgeDoc can use an LDAP server to authenticate users. As HedgeDoc supports multiple LDAP servers, you first need to tell HedgeDoc the servers you want to configure (`HD_AUTH_LDAPS`), and then you need to provide a few variables for those LDAP servers depending on how you want to use them.
+Each of those variables will contain the given name for this LDAP server. For example if you named your ldap server `MY_LDAP` all variables for this server will start with `HD_AUTH_LDAP_MY_LDAP`.
+
+| environment variable                       | default              | example                                            | description                                                                                                     |
+|--------------------------------------------|----------------------|----------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| `HD_AUTH_LDAPS`                            | -                    | `MY_LDAP`                                          | A comma-seperated list of LDAP servers HedgeDoc should use.                                                     |
+| `HD_AUTH_LDAP_$NAME_PROVIDER_NAME`         | `LDAP`               | `My LDAP`                                          | The name for the ldap server `$NAME`, that is shown in the UI of HegdeDoc.                                      |
+| `HD_AUTH_LDAP_$NAME_URL`                   | -                    | `https://ldap.example.com`                         | The url with which the LDAP server `$NAME` can be accessed.                                                     |
```

Comment:
LDAP urls don't use HTTP(s). They use `ldap://`, `ldaps://` or `ldapi://`

Reply:
Good point maybe we want to add this to the joi config for the LDAP url as well? 🤔